client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -1993,18 +1999,22 @@ export default {
     },
 
     handleEditRecordAction (recordID) {
```

Comment:
Inline actions shouldn't use the recordDisplayOption. They are separate from clicks on record/add button
Only the on record click event should use those (and the option you added)